pinot-common/pom.xml

```diff
@@ -161,6 +161,10 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpmime</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Thanks for the great effort of this migration. After introducing httpclient5, do we still need to keep the dependency for http4 libraries?